src/index-theme.html

```diff
@@ -0,0 +1,206 @@
+<html>
+  <head>
+    <title>ECP Demo</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="app">
+      <div class="description">
+        <h1>Change theming</h1>
+        <p>
+          On ECP, it is possible add some visual customisation. This feature
+          will allow you to match your corporate and ECP colors. You can change
+          your theme as many time as you want thanks to the render function.
+        </p>
+        <p>The first step you need to do is integrate ECP using the sdk:</p>
+        <pre>
+          <code>
+              &lt;div class="ecp-frame symphony-ecm"&gt;
+              &lt;/div&gt;
+          </code>
+        </pre>
+        <p>and add the script:</p>
+        <pre>
+          <code>
+              &lt;script id="symphony-ecm-sdk" data-onload="onSdkLoaded" render="explicit" src="{ECP-SDK-URL}"&gt;&lt;/script&gt;
+          </code>
+        </pre>
+        <p>
+          Notice the <code>data-onload</code> and
+          <code>render="explicit"</code> attributes to tell the sdk you want to
+          run ECP in explicit mode.
+        </p>
+        <p>
+          Once the script is loaded it will call your previously defined
+          <code>onSdkLoaded</code> function passed in data-onload attribute.
+          Here we simply use it to call the sdk <code>render</code> function.
+        </p>
+        <pre>
+          <code>
+            window.onload = () =&gt; {
+              window.symphony.render('symphony-ecm', {
+                streamId, 
+                theme: {
+                  primary: "#ff00dd",
+                  secondary: "#ac0202",
+                  accent: "#F7CA3B",
+                  success: "#2EAA35",
+                  error: "#DE342E",
+                  background: "#ffb8b8",
+                  surface: "#ffe0e0",
+                  text: "#000000",
+                  textPrimary: "#FFFFFF",
+                  textSecondary: "#FFFFFF",
+                  textAccent: "#17181B",
+                  textSuccess: "#FFFFFF",
+                  textError: "#FFFFFF",
+                },
+              })
+            }
+          </code>
+        </pre>
+        <p>
+          Once ECP is rendered, you can easily switch theme without reloading
+          the whole aplpication with the <code>updateTheme</code> function. This
+          is what we do below if the sdk is already initialized (ie the form has
+          already been submitted once).
+        </p>
+        <p>There is 2 types of theme options :</p>
+        <ul>
+          <li>
+            Mandatory :
```

Comment:
Could be nice to have the types of each or a link to the doc with the types. 
The mandatory colors are just hex colors in string while the shades are maps of colors.